specification/marketplacecatalog/resource-manager/Products/preview/2022-09-01-preview/products.json

```diff
@@ -0,0 +1,1411 @@
+            "type": "boolean",
+            "name": "lookupOfferInTenantLevel",
+            "in": "query",
+            "description": "Optional - in case you want to check against tenant private audience also",
+            "default": false,
+            "x-nullable": false
+          },
+          {
+            "type": "boolean",
+            "name": "includeHiddenPlans",
+            "in": "query",
+            "description": "Denotes whether to include hidden plans",
+            "default": false,
+            "x-nullable": false
+          },
+          {
+            "type": "string",
+            "name": "x-ms-client-object-id",
+            "x-originalName": "userObjectId",
+            "in": "header",
+            "description": "AAD user ID",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "x-ms-client-alt-sec-id",
+            "x-originalName": "altSecId",
+            "in": "header",
+            "description": "Alternate Security ID - relevant for MSA user",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a single product directly from BigCat, by subscription as audience",
+            "schema": {
+              "$ref": "#/definitions/ProductSummary"
+            }
+          },
+          "404": {
+            "x-nullable": false,
+            "x-ms-error-response": true,
+            "description": "In case the product does not exist, does not exist in the given market or is not available for the given audience",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "x-ms-error-response": true,
+            "description": "In case the request is malformed",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "401": {
+            "x-nullable": false,
+            "x-ms-error-response": true,
+            "description": "In case the provided authorization token is not in allowed list",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "403": {
+            "x-nullable": false,
+            "x-ms-error-response": true,
+            "description": "In case the provided authorization token validation has failed",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateProduct_Get": {
+            "$ref": "./examples/SinglePrivateProduct_Get.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/products/{id}": {
```

Comment:
>"/providers/Microsoft.Marketplace/products/{id}" [](http://example.com/codeflow?start=4&length=48)

Didn't get, as how products resources are there at subscription as well as Tenant scope. Do you have any writeup to understand this RP design?